### PR TITLE
Symorton/claim parsing

### PR DESCRIPTION
**Pull request by symorton**
**Problem:**
We need to parse the OIDC4VCIVerifiedId claims.

**Solution:**
Implement getClaims() to get localized claims.

**Validation:**
Unit tests pass.

**Type of change:**
- [X] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [X] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift [hunk 1]
```diff
@@ -80,10 +84,39 @@ struct OpenID4VCIVerifiedId: InternalVerifiedId
...
     }
+    
+    private func createVerifiedIdClaim(claimReference: String, claimValue: Any) -> VerifiedIdClaim
+    {
+        if let claimDefinitions = configuration.credential_definition?.credential_subject,
+           let claimDisplayDefinitions = claimDefinitions["vc.credentialSubject.\(claimReference)"],
+           let localizedDefinition = claimDisplayDefinitions.getPreferredLocalizedDisplayDefinition(),
+           let label = localizedDefinition.name
+        {
+            return VerifiedIdClaim(id: label,
```
Comment (nithyaganeshng) [inline WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift]: Isn't type needed to be bubbled up here to format the claim in UI?

File: WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift [hunk 2]
```diff
@@ -80,10 +84,39 @@ struct OpenID4VCIVerifiedId: InternalVerifiedId
...
+                                   value: claimValue)
+            
+        }
+        else
+        {
+            return VerifiedIdClaim(id: claimReference,
```
Comment (nithyaganeshng) [inline WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift]: Same as above.

File: WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift (hunk 1 above)
Comment (symorton) [reply WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift]: Updated.

File: WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift (hunk 2 above)
Comment (symorton) [reply WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift]: Updated.

**Review by nithyaganeshng — APPROVED** [review]

**Pull request merged**